src/treeview_commands.ml

```diff
@@ -73,7 +73,32 @@ let jump_item =
   Vscode.TreeItem.set_command item command;
   item
 
-let items = [ select_sandbox_item; terminal_item; construct_item; jump_item ]
+let navigate_holes_item =
+  let icon = `ThemeIcon (Vscode.ThemeIcon.make ~id:"arrow-up" ()) in
```

Comment:
I think the `activate-breakpoints` icon would symbolize holes nicely, what do you think ?

Reply:
The icon looks great. It's literally holes :)